test/integration_test.go

```diff
@@ -4136,3 +4136,20 @@ func TestTfPath(t *testing.T) {
 
 	assert.Regexp(t, "(?i)(terraform|opentofu)", stdout+stderr)
 }
+
+func TestVersionIsInvokedOnlyOnce(t *testing.T) {
```

Comment:
We should include a version where at least two fire from `.tool-version` files, etc.